src/pages/signin/TermsAndLicenses/TermsWithLicenses.js

```diff
@@ -3,35 +3,49 @@ import {Text, View} from 'react-native';
 import styles from '../../../styles/styles';
 import CONST from '../../../CONST';
 import TextLink from '../../../components/TextLink';
-import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import withLocalize, {
+    withLocalizePropTypes,
+} from '../../../components/withLocalize';
+import LogoWordmark from '../../../../assets/images/expensify-wordmark.svg';
 
 const TermsWithLicenses = ({translate}) => (
-    <View style={[styles.mt6, styles.flexRow, styles.flexWrap]}>
-        <Text style={[styles.loginTermsText]}>
+    <View>
+        <View style={[styles.mt1, styles.alignItemsCenter, styles.mb3]}>
+            <LogoWordmark height={30} width={80} />
+        </View>
+        <Text style={[styles.textAlignCenter, styles.loginTermsText]}>
             {translate('termsOfUse.phrase1')}
             {' '}
-        </Text>
-        <TextLink style={[styles.loginTermsText]} href={CONST.TERMS_URL}>
-            {translate('termsOfUse.phrase2')}
-        </TextLink>
-        <Text style={[styles.loginTermsText]}>
+            <TextLink
+                style={[styles.loginTermsText, styles.termsLink]}
+                href={CONST.TERMS_URL}
+            >
+                {translate('termsOfUse.phrase2')}
+            </TextLink>
             {' '}
             {translate('termsOfUse.phrase3')}
             {' '}
-        </Text>
-        <TextLink style={[styles.loginTermsText]} href={CONST.PRIVACY_URL}>
-            {translate('termsOfUse.phrase4')}
-        </TextLink>
-        <Text style={[styles.loginTermsText]}>.</Text>
-        <Text>
-            <Text style={[styles.loginTermsText]}>
-                {translate('termsOfUse.phrase5')}
-                {' '}
-            </Text>
-            <TextLink style={[styles.loginTermsText]} href={CONST.LICENSES_URL}>
-                {translate('termsOfUse.phrase6')}
+            <TextLink
+                style={[styles.loginTermsText, styles.termsLink]}
+                href={CONST.PRIVACY_URL}
+            >
+                {translate('termsOfUse.phrase4')}
+            </TextLink>
+            .
+            {'\n'}
+            {translate('termsOfUse.phrase5')}
+            {' '}
+
+
```

Comment:
Remove unnecessary spaces here

Reply:
In android, the behaviour is a bit different therefore I have used Platform.select

Reply:
Please don't use `Platform.select` - use [platform-specific file extensions](https://github.com/Expensify/Expensify.cash#platform-specific-file-extensions) instead